spec/lib/nacre/product_collection_spec.rb

```diff
@@ -39,7 +39,11 @@
     let(:get_options) do
       {
         search_url: product_service_url,
-        ids: range
+        ids: range,
+        options: [
+          'customFields',
+          'nullCustomFields'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.